src/forms/TextInput.jsx

```diff
@@ -5,79 +5,62 @@ import cx from 'classnames';
 /**
  * @module TextInput
  */
-class TextInput extends React.Component {
+function TextInput(props) {
```

Comment:
I was thinking the same thing. Torn between es6 syntax and a named function. @ivy @eilinora